omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -129,11 +139,39 @@
 				
 				<prop key="admin/provider/providerAttributeType.list">providerAttributeTypeList</prop>
 				<prop key="admin/provider/providerAttributeType.form">providerAttributeTypeForm</prop>	
-				
+
  			</props>
 		</property>
 	</bean>	
 
+ 	<!-- index pages - begin -->  
+	
+	<!-- admin -->
+	<mvc:view-controller view-name="module/legacyui/admin/index" path="admin/index.htm"/>				
```

Comment:
Do as the ticket says. Index pages only. That will be another ticket. :)

On Tue, Nov 3, 2015 at 12:24 AM, jdegraft <notifications@github.com> wrote:

> In omod/src/main/resources/webModuleApplicationContext.xml
> <https://github.com/openmrs/openmrs-module-legacyui/pull/14#discussion_r43684566>
> :
>
> >   			</props>
> >  		</property>
> >  	</bean>	
> >
> > + 	<!-- index pages - begin -->
> > +	
> > +	<!-- admin -->
> > +	<mvc:view-controller view-name="module/legacyui/admin/index" path="admin/index.htm"/>				
>
> One final question.
> Do I restrict these changes to index pages or in general?
> For instance if there is a reference to /admin/patientForm.jsp do I change
> it /module/legacyui/patientForm.jsp or leave that to be done in another
> ticket?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openmrs/openmrs-module-legacyui/pull/14/files#r43684566>
> .
>



-- 
If we keep uppermost in our minds the unkind and unjust acts of others, we
shall find it impossible to love them as Christ has loved us; but if our
thoughts dwell upon the wondrous love and pity of Christ for us, the same
spirit will flow out to others.